core/src/main/resources/lib/layout/breadcrumbs.js

```diff
@@ -156,20 +156,22 @@ window.breadcrumbs = (function () {
     }
 
     // ignore the currently pending call
-    if (xhr) {
-      xhr.options.onComplete = function () {};
+    if (controller) {
+      controller.abort();
     }
 
     if (e.items) {
       // use what's already loaded
       showMenu(e.items());
     } else {
       // fetch menu on demand
-      xhr = new Ajax.Request(
-        combinePath(e.getAttribute("href"), contextMenuUrl),
-        {
-          onComplete: function (x) {
-            var items = x.responseText.evalJSON().items;
+      controller = new AbortController();
+      let { signal } = controller;
+      fetch(combinePath(e.getAttribute("href"), contextMenuUrl), { signal })
```

Comment:
HtmlUnit appears to lack native support for fetch: https://github.com/jenkinsci/jenkins/pull/7731#pullrequestreview-1368116484

Per [the documentation](https://htmlunit.sourceforge.io/webclient.html#Polyfills), "starting with version 2.59.0 HtmlUnit supports the integration of polyfills":

```java
webClient.getOptions().setFetchPolyfillEnabled(true);
```

If core is to begin relying on the Fetch API, then the `WebClient` returned by `createWebClient()` in the Jenkins Test Harness (JTH) needs to have this option set to `true`. Otherwise, every consumer of core (which includes every plugin) will need to repeat the setting of this option to `true` when testing a page that invokes the Fetch API.

Reply:
https://github.com/jenkinsci/jenkins-test-harness/pull/567